.devcontainer/cpp/test/testsuite.bats

```diff
@@ -205,3 +205,15 @@ function build_and_run_with_sanitizers() {
   assert_failure
   assert_output --partial "runtime error: load of null pointer"
 }
+
+@test "using clang-uml to generate uml class diagram" {
+  run cmake --preset clang
+  assert_success
+
+  run cmake --build --preset clang-uml
+  assert_success
```

Comment:
Same as above. Remove the `run` and `assert_success`.

Reply:
Done